oneview/resource_enclosure_group.go

```diff
@@ -279,22 +304,41 @@ func resourceEnclosureGroupRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("ambient_temperature_mode", enclosureGroup.AmbientTemperatureMode)
 	d.Set("associated_logical_interconnect_groups", enclosureGroup.AssociatedLogicalInterconnectGroups)
 	d.Set("category", enclosureGroup.Category)
-	d.Set("description", enclosureGroup.Description)
 	d.Set("eTag", enclosureGroup.ETAG)
 	d.Set("enclosure_count", enclosureGroup.EnclosureCount)
 	d.Set("enclosure_type_uri", enclosureGroup.EnclosureTypeUri.String())
-	d.Set("initial_scope_uris", enclosureGroup.InitialScopeUris)
 	d.Set("interconnect_bay_mapping_count", enclosureGroup.InterconnectBayMappingCount)
 	d.Set("interconnect_bay_mappings", enclosureGroup.InterconnectBayMappings)
 	d.Set("ip_addressing_mode", enclosureGroup.IpAddressingMode)
 	d.Set("ip_range_uris", enclosureGroup.IpRangeUris)
+	d.Set("ipv6_addressing_mode", enclosureGroup.Ipv6AddressingMode)
+	d.Set("ipv6_range_uris", enclosureGroup.Ipv6RangeUris)
 	d.Set("name", enclosureGroup.Name)
 
+	interconnectBayMap := make([]map[string]interface{}, 0, 1)
+	for i := 0; i < len(enclosureGroup.InterconnectBayMappings); i++ {
+		liguri := enclosureGroup.InterconnectBayMappings[i].LogicalInterconnectGroupUri
+
+		encLIG, err := config.ovClient.GetLogicalInterconnectGroupByUri(liguri)
+		if err != nil || encLIG.Name == "" {
+			d.SetId("")
+			return nil
```

Comment:
please return an error statementif no LIG is found.

Reply:
Done